scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
@bfredl I do not like the idea of invoking *anything* with top-level Makefile, especially since now even `make clint` is going to pull in dependencies. And if I am debugging e.g. declarations generator I would not want to run it on *all* files, so gen_xx is immediately out even if available from build/Makefile, defined in src/nvim/CMakeLists.txt and only proxied from top-level Makefile like most other its targets are. Additionally where generated files are placed is much of a tab-completion waste then extra bin: Vim `:e` is nearly not as good at completion as zsh, manually specifying the target file to a reachable location is better.